standards/out-of-hours.md

```diff
@@ -0,0 +1,60 @@
+
+## Determining if your service needs support out of hours
+
+Not all services need out of hours support. When considering the need for out of hours support for your service, you first need to understand the risk if the service were to go down when no-one is around to fix it.
+
+The kinds of risk that might suggest out of hours support is necessary include:
+
+- Risk to someone's life
+- The MOJ's ability to provide critical, time-sensitive justice services
+- Impact to staff's and suppliers' ability to do carry out their job safely and to the best of their ability
+- Impairing citizens', suppliers', and partners' ability to interact with MOJ services in a way that meets their needs
+
+These risks should be considered alongside your service's robustness.
+
+For example, if your service can recover from most classes of issue itself, and downtime means minor impairment to a small group of people whose task is not time-sensitive, out of hours support may not be necessary. If, however, a problem with your service (whether outages, security breaches, or something else) out of hours presents a genuine risk to someone's life, it _may_ be appropriate to provide out of hours support around the clock, depending on the robustness of the system.
+
+## Assessing your team's ability to provide out of hours support
+
+To run a sustainable out of hours rota, you must first ensure that either:
+
+- You have enough staff (including contractors) to support the service out of hours sustainably (i.e. without risking burnout), and have the ability to pay them for their time supporting the service
+
+Or:
+
+- You have budget to pay a third party to provide your support sustainably, and have the ability to train them sufficiently before they are expected to start supporting it.
+
+If you are unable to do either of these, you cannot support your service out of hours without risking people's health and well-being. If that is the case, you should escalate this risk and not provide support until you are in a position to do one or other of those.
+
+## Managing a sustainable out of hours rota
+
+Being on call out of hours impacts people in different ways. Some people (especially people new to being on call) find the idea of being on call inherently stressful, and put their lives on hold while they are on call. Others are able to integrate being on call into their lives, happily taking their laptop and phone wherever they go and moderating their activities to allow that.
+
+Either kind of person, and all of those on the spectrum between, however, can easily burn out if a few core criteria are not met:
+
+- No one person is on call more than 50% of the time, and ideally no more than 30% of the time.
```

Comment:
I know that you're trying to define a baseline here, but I'm uncomfortable with having numbers on this without also considering the impact of expected response time on the sustainability of being on call frequently.

Even if you're happy to take your laptop and phone everywhere and don't find the idea of on-call stressful, there's still a burden in planning your activities so that you have fairly consistent phone signal, and are near enough to somewhere where you could plausibly sit down, get your laptop out, get online and concentrate on resolving an incident. The restrictive impact on people's lives of needing to be able to do that within 15 mins vs 4 hours (for example) is very different, and can increase the burden of being on call significantly and make an otherwise acceptable rota unsustainable.

Reply:
I think if a person is on call the implication is that their activities are necessarily restricted in order that they can be available. To my mind this is a basic expectation of being on call.

Reply:
It might make sense to split this list into two sections:

- the core criteria, avoiding any specific numbers but perhaps posing questions instead (eg.: what percentage of the on-call rota would your team members be comfortable covering on a regular basis?) 

- a more specific section, an example of sorts, that addresses the questions posed by presenting an actual setup, eg.: based on a team of eight, this is what you could be doing because it has the following benefits.

This way, we can document the various concerns that come with on-call rota but leave it to the teams and the individuals to come up with the specifics and decide what works best for their situation.

Additionally, it might also make sense to have some feedback channel open. Not everyone will feel equally comfortable to voice their concerns about a team's existing on-call schedule, especially if they are new members of that team.

Reply:
@jennyd That definitely is something that people running a rota need to consider, but given that people are running on call rotas with 1-2 people, I'm keen to have a baseline. I've added another point to try to address some of the issues about intensity of being on call that you raise, though.

Reply:
Is this conversation resolved? I notice the guidance now says:

> You have enough staff (including contractors) to support the service out of hours sustainably (realistically this means at least six people, ideally eight), and have the ability to pay them for their time supporting the service

I strongly agree with this and this is what I would want to emphasise rather than this 50% number. If you have 8 people on the rota you will not come close to somebody being on call 30% of their time.

This should absolutely not be left to individual teams, who will just say "well, we only have one person who knows about this, so they will have to be on call" or even worse, expect people to be effectively on call without acknowledging it or compensating people.

Reply:
@MatMoore That line now reads as follows: 

> - No one person is on call 50% of the time or more, and ideally no more than 30% of the time. This, along with the next criteria, mean that you will need at least six people, and ideally eight, to run a rota sustainably.
> - At least two people are on call at any one time, and know which of them will be contacted first (primary on-call) and second (secondary on-call).

I'm keen to keep the percentages in too, just to make sure we don't have teams running rotas where one person is primary 100% of the time while 7 others rotate through secondary (for example).

What do you think? Does that address your concerns?